dockerfiles/Dockerfile.arm32v7

```diff
@@ -14,7 +14,8 @@ RUN install_packages \
     python3-pip \
     python3-dev \
     git \
-    tar
+    tar \
+    libatlas-base-dev
```

Comment:
Why atlas is needed?

Reply:
Without this, numpy failed to load complaining libaltlas...so is missing.